src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using CppSharp.AST;
+
+namespace CppSharp.Passes
+{
+    public class CleanCommentsPass : TranslationUnitPass
+    { 
+        public override bool VisitDeclaration(Declaration decl)
+        {
+            if (decl.Comment != null)
+            {
+                var summaryPara = (ParagraphComment)decl.Comment.FullComment.Blocks[0];
+
+                foreach (AST.TextComment item in summaryPara.Content)
+                {
+                    if (item.Text.Contains("<") || item.Text.Contains(">"))
+                        item.Text = item.Text.TrimStart('<', '>');
+                    if (item.Text.Contains("summary"))
+                        item.Text = "";
+                        
+                }
+                decl.Comment.FullComment.Blocks[0] = summaryPara;
+            }
+            return base.VisitDeclaration(decl);
```

Comment:
As you can see in other passes, they usually start with:

    if (!base_call)
        return false;

This ensures already visited declarations are skipped. Do it here too.